contracts/SpokePool.sol

```diff
@@ -955,18 +955,17 @@ abstract contract SpokePool is
         nonReentrant
         unpausedFills
     {
+        // Exclusivity deadline is inclusive and is the latest timestamp that the exclusive relayer has sole right
+        // to fill the relay.
         // solhint-disable-next-line not-rely-on-time
-        require(relayData.fillDeadline >= block.timestamp, "Fill deadline expired");
-        // solhint-disable-next-line not-rely-on-time
-        require(
-            relayData.exclusivityDeadline >= block.timestamp && relayData.exclusiveRelayer == msg.sender,
-            "Not exclusive relayer"
-        );
+        if (relayData.exclusivityDeadline >= block.timestamp && relayData.exclusiveRelayer != msg.sender) {
```

Comment:
Is it cheaper to reverse these, since the vast majority of fills will fall within the exclusivity deadline? It should save evaluating the 2nd statement in most cases.

```suggestion
        if (relayData.exclusiveRelayer != msg.sender && relayData.exclusivityDeadline >= block.timestamp) {
```

Reply:
that's a good point